### PR TITLE
Update Blazor Description

### LINKED ISSUE #14641: Blazor description is wrong
The description of Blazor on this page is incorrect - Server side Blazor has nothing to do with WebAssembly.

Suggested edit:

> ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with C# and Razor on the server and for the UI to be interactively connected to the browser in real-time using a persistent SignalR connection. For more information about Blazor, see Get started with Blazor.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1ad8df5d-9295-d659-a561-ec9cabd5b61f
* Version Independent ID: 7ee71103-d997-140e-2d35-d9f2a3913e09
* Content: [Choose between traditional web apps and single page apps](https://docs.microsoft.com/en-us/dotnet/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps)
* Content Source: [docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md](https://github.com/dotnet/docs/blob/master/docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md)
* Product: **dotnet-architecture**
* Technology: **modern-web-apps-azure**
* GitHub Login: @ardalis
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Fixes and updates Blazor description

Fixes #14641
https://github.com/dotnet/docs/issues/14641

### CODE REVIEW

**Review by nishanil — APPROVED** [review]
Looks good.

File: docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ Improvements in user experience made possible by the SPA approach must be weighe
 
 ## Blazor
 
-ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with Razor on the server and for this code to be delivered to the browser and executed client-side using [WebAssembly](https://webassembly.org/). Blazor server-side is available now with ASP.NET Core 3.0 or later. Blazor client-side should be available in 2020.
+ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with C# and Razor on the server and for the UI to be interactively connected to the browser in real-time using a persistent SignalR connection.
```
Comment (guardrex) [inline docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md:38]: FYI for future reference (or a patch PR) ... Use "Blazor Server" ... "Blazor server-side" is an older product name that we don't user any longer.

File: docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ Improvements in user experience made possible by the SPA approach must be weighe
...
+
+Blazor WebAssembly introduces another option for Blazor apps, allowing them to run in the browser using WebAssembly. Because it's real .NET running on WebAssembly, you can re-use code and libraries from server-side parts of your application.
```
Comment (guardrex) [inline docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md:40]: Note in passing ... A Blazor WebAssembly app (standalone) doesn't have "server-side parts," so this description is a little off. The phrase "server-side parts of your application" implies that such apps always have "server-side" parts. However, I understand that readers will need to get into the hosting model details in the Blazor docs to understand it further. There's no important reason to spend a lot of time here with the concepts.

**Comment by nishanil** [comment]
@ardalis Can you make a note of this and send a separate PR please?

**Comment by ardalis** [comment]
You bet.

**Pull request merged**